### ISSUE TITLE
Remove Virus from Version 1.2.164

### ISSUE THREAD

**Issue by marcobsidian** [pop4959/Chunky#302]
Expected Behavior:
- Chunky (Fabric) v1.2.146 is not a virus

Observed Behavior:
- Chunky (Fabric) v1.2.146 is a virus

Change Request:
- [ ] Remove any viruses from the mod as I would not like to get my identity stolen while caving for coal and iron

Additional Information:
![image](https://github.com/pop4959/Chunky/assets/84506368/f8df7e3c-c73a-4de8-84dd-54906677c787)

**Comment by marcobsidian** [comment]
Even further information:
![image](https://github.com/pop4959/Chunky/assets/84506368/464a464f-16b3-46c9-b9df-c8ebdd684fd5)

**Comment by pop4959** [comment]
There isn't a virus, chunky has recently been flagged as a false positive by some antiviruses (apparently due to the trim feature). We're working on getting these false positives removed, but until then, you can safely ignore that warning.

Will leave this up as a PSA until this is resolved since I'm sure others will have the same question.

**Comment by pop4959** [comment]
As an update, it looks like false positives have been removed for some versions of chunky, although other versions remain affected. Still working on getting those fixed up.

**Comment by pop4959** [comment]
Most antiviruses appear to have updated and removed the false positive for chunky. The remaining ones are likely to clear within a few days. I'll close this later in the week if I don't see any other reports.

**Comment by pop4959** [comment]
Closing as this appears fully resolved.

**Comment by Randomsito** [comment]
DeepInstinct in Virus total flags this as malicious
https://www.virustotal.com/gui/file/6662389bb4ed0b12b7380feabbfdea8325312d5a7979e8c25fb1edbe38f09509

**Comment by pop4959** [comment]
> DeepInstinct in Virus total flags this as malicious (forge 1.19.2 with chunky latest version) https://www.virustotal.com/gui/file/6662389bb4ed0b12b7380feabbfdea8325312d5a7979e8c25fb1edbe38f09509

Yeah that one seems to be taking longer to clear the false positive. I may take another look but I didn't see any obvious contact form like with BitDefender (and every other antivirus updated after BitDefender updated their checks).

Regardless, just note that this is indeed a false positive and nothing to need to worry about.
